modules/ui/entity_editor.js

```diff
@@ -29,9 +30,95 @@ export function uiEntityEditor(context) {
 
     var _sections;
 
-    function entityEditor(selection) {
 
-        var combinedTags = utilCombinedTags(_entityIDs, context.graph());
+    // Returns a single object containing the tags of all the given entities.
```

Comment:
Okay, I'll bite. Why the refactor of this method from the `util` methods to this file? Not used anyplace else?

Reply:
Yeah, not used anywhere else.. @vanreece and I found this one and were struggling to SDKify it - we decided to just put it back into the enity_editor (only place it's used).